pkg/cloudcost/integration.go

```diff
@@ -57,15 +57,24 @@ func GetIntegrationFromConfig(kc cloud.KeyedConfig) CloudCostIntegration {
 	case *azure.StorageConnection:
 		return &azure.AzureStorageIntegration{
 			AzureStorageBillingParser: azure.AzureStorageBillingParser{
-				StorageConnection: *keyedConfig,
+				StorageConnection: azure.StorageConnection{
+					StorageConfiguration: keyedConfig.StorageConfiguration},
```

Comment:
refactor to const? these all look identical?

Reply:
Not really sure how this would be a const, I tried to make it a multi value case but that is invalid even though they all have a `StorageConfiguration` property. The only thing I could really see tightening this up is a constructor method